src/main/java/user/requestconnect/RequestConnectionController.java

```diff
@@ -21,15 +25,30 @@ public class RequestConnectionController {
     @Autowired
     SimpUserRegistry userRegistry;
 
+    /**
+     * Get target users that the principal wishes to connect with from userRegistry
+     */
     public void findThem() {
         System.out.println(userRegistry.getUsers());
     }
+
+    /**
+     *
+     * @param requestConnectionInteractor is the use case interactor
+     * @param simpMessagingTemplate provides methods for sending messages to users
+     */
     @Autowired
     public RequestConnectionController(IRequestConnectionInput requestConnectionInteractor,
                                        SimpMessagingTemplate simpMessagingTemplate) {
         this.requestConnectionInteractor = requestConnectionInteractor;
         this.simpMessagingTemplate = simpMessagingTemplate;
     }
+
+    /**
+     * The requestConnection method returns a reponseModel depending on the ServerStatus
```

Comment:
First word should be the action, say "Returns (or creates) a responseModel depedning on the ServerStatus"